.prettierrc.json

```diff
@@ -1,7 +1,7 @@
-{
-  "printWidth": 120,
-  "singleQuote": true,
-  "arrowParens": "avoid",
-  "semi": true,
-  "endOfLine": "crlf"
-}
\ No newline at end of file
+{

```

Comment:
Dlaczego zmiana rozszerzenia?

Reply:
Byłem przekonany, że mi prettier przez to nie działa :D